src/components/homePage/Contributors.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div id="contributors">
+    <div>
+      <h2 style="display: inline;">Created with love in &nbsp;</h2>
+      <img style="display: inline;" src="../../assets/Canada.png" />
+      <h2 style="display: inline;">&nbsp; by</h2>
+    </div>
+    <div class="gallery">
+      <div class="parent" v-for="(member,index) in members" :key="member.name">
+        <a @mouseover="open(index)" style="text-decoration: none;">
```

Comment:
I should've clarified my statement earlier: if the hover ends, the index should be null'd again so that the final thing can disappear. 

As well, a fade in/out transition would be nice, but totally optional :)